manticore/utils/install_helper.py

```diff
@@ -0,0 +1,109 @@
+import subprocess
+import sys
+import pkg_resources
+
+
+REQUIREMENTS_TO_IMPORTS = {
```

Comment:
This thing might be discussable. Effectively, the evm and native requirements are duplicated here and in `setup.py`. This is because we need to try import all of them to ensure that either evm or native dependencies are installed (and this dict maps pypi package name to its import).

So... in theory we could provide the evm/native package requirements JUST HERE and import it in setup.py, but I am not sure if we want this.

On the other hand I already import this stuff in setup.py to ensure that those lists stay synchronised. 

@mossberg @feliam @yan Please provide feedback; is that fine?